silx/gui/plot/ImageView.py

```diff
@@ -656,13 +610,13 @@ def setRadarView(self, radarView):
         :param RadarView radarView: Widget subclassing RadarView to replace
                                     the lower right corner widget.
         """
-        self._radarView.visibleRectDragged.disconnect(self._radarViewCB)
+        if self._radarView is not None:
```

Comment:
radarView can't be `None`, else `self.centralWidget().layout().addWidget(self._radarView, 1, 1)` would fail.